nemo/collections/asr/metrics/rnnt_wer_bpe.py

```diff
@@ -157,6 +164,80 @@ def decode_ids_to_tokens(self, tokens: List[int]) -> List[str]:
         token_list = self.tokenizer.ids_to_tokens(tokens)
         return token_list
 
+    def decode_tokens_to_lang(self, tokens: List[int]) -> str:
+        """
+        Compute the most likely language ID (LID) string given the tokens.
+
+        Args:
+            tokens: List of int representing the token ids.
+
+        Returns:
+            A decoded LID string.
+        """
+        lang = self.tokenizer.ids_to_lang(tokens)
+        return lang
+
+    def decode_ids_to_langs(self, tokens: List[int]) -> List[str]:
+        """
+        Decode a token id list into language ID (LID) list.
+
+        Args:
+            tokens: List of int representing the token ids.
+
+        Returns:
+            A list of decoded LIDS.
+        """
+        lang_list = self.tokenizer.ids_to_text_and_langs(tokens)
+        return lang_list
+
+    def decode_ids_to_words_and_langs(self, tokens: List[int]) -> List[str]:
```

Comment:
Why is this necessary ? We won't have one method for every single combination like this. Try to decompose operations and then build step by step

Reply:
fixed